hooks/ecosystem-conversion-utils.ts

```diff
@@ -21,9 +21,6 @@ async function fetchMembers() {
       });
     });
 
-    // Sort ecosystem projects alphabetically
-    membersArray.sort((a, b) => a.name?.localeCompare(b.name));
-
```

Comment:
This is actually useful. It sorts the projects alphabetically so that we can avoid having to sort them again in the **pages/ecosystem.vue** file when the user selects the "Name" order option. It reduces the time complexity from **O(n * log n)** to **O(1)**.